server.py

```diff
@@ -41,11 +41,17 @@ def create_application():
 @APP.route(f'{ROUTE_PREFIX}/', methods=['GET'], strict_slashes=False)
 def get_root():
     """Root Endpoint for 3scale."""
```

Comment:
can you add a test?

Reply:
Sure, although it looks like in order to write a test for `APP` routes, we would have to make a change in the code where we set `WORKERS`

For the test, I will be making a follow-up PR